Bottom: da622ffbf65758283bb538088e9db61713af8e01
Top:    045d63c04560d6e9c122c4659f38398962d6ef8b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 15:54:17 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index b2add35..ff94ccf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
